dali/pipeline/pipeline.cc

```diff
@@ -537,8 +554,8 @@ void Pipeline::Build(std::vector<PipelineOutputDesc> output_descs) {
       DALI_ENFORCE(device_id_ != CPU_ONLY_DEVICE_ID,
                    make_string(
                      "Cannot move the data node ", name, " to the GPU "
-                     "in a CPU-only pipeline. The `device_id` parameter "
-                     "is set to `CPU_ONLY_DEVICE_ID`. Set `device_id` "
+                     "in a CPU-only pipeline. The 'device_id' parameter "
+                     "is set to `CPU_ONLY_DEVICE_ID`. Set 'device_id' "
```

Comment:
This is an interesting case. There's no `CPU_ONLY_DEVICE_ID` in Python. It's `None`.
Perhaps we should replace it in the Python bindings?

Reply:
We had this for a while, I think it would be nice to fix, but the solution would probably warrant a separate PR.